pkg/cmd/openshift-tests/run/flags.go

```diff
@@ -75,10 +76,14 @@ func (f *RunSuiteFlags) SetIOStreams(streams genericclioptions.IOStreams) {
 
 func (f *RunSuiteFlags) ToOptions(args []string) (*RunSuiteOptions, error) {
 	adminRESTConfig, err := kubeconfig.GetStaticRESTConfig()
-	if err != nil {
+	if err != nil && !f.GinkgoRunSuiteOptions.DryRun {
 		return nil, err
 	}
 
+	if err != nil && f.GinkgoRunSuiteOptions.DryRun {
+		adminRESTConfig = &rest.Config{}
+	}
+
```

Comment:
```suggestion
	switch {
	case err != nil && f.GinkgoRunSuiteOptions.DryRun:
		fmt.Fprintf(f.ErrOut, "Unable to get admin rest config, skipping apigroup check in the dry-run mode: %v\n", err)
		adminRESTConfig = &rest.Config{}
	case err != nil && !f.GinkgoRunSuiteOptions.DryRun:
		return nil, fmt.Errorf("unable to admin rest config, skipping apigroup check in the dry-run mode: %w", err)
```

Reply:
@vrutkovs Isn't it a wrong message in line 84 that you suggested?
The condition in line 83 above says !DryRun, whereas the error message has the text `skipping apigroup check in the dry-run mode`
I suppose the Error message at https://github.com/openshift/origin/blob/14db7484bdd1a1e0d97f1d806905ccd04a237778/pkg/clioptions/suiteselection/suite_flags.go#L114-L115 is also wrong.

Reply:
Ah, right, sorry, I copypasted it. Feel free to adjust the error message when applying the suggestion and squash commits